snapcraft/internal/project_loader/_plugs_config.py

```diff
@@ -0,0 +1,74 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2019 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+from typing import Dict, Set  # noqa: F401
+
+from snapcraft.internal.repo.snaps import SnapPackage
+
+logger = logging.getLogger(__name__)
+
+
+class Plug:
```

Comment:
from the first commit, something like this I guess

Reply:
After responding below, now I get this.  And I do believe we are in agreement :)